### PR TITLE
vuefy dataset information

### PR DESCRIPTION
**Pull request by OlegZharkov**
Another step towards removing `show_params.mako` and transfering that page to Vue.
image

### CODE REVIEW

File: client/src/components/DatasetInformation/mount.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Endpoint for mounting job metrics from non-Vue environment.
+ */
+import $ from "jquery";
```
Comment (dannon) [inline client/src/components/DatasetInformation/mount.js]: Could you do this without jquery please?

Comment (OlegZharkov) [reply client/src/components/DatasetInformation/mount.js]: It is temporary code, this `mount.js` file will be gone as soon as we convert `show_params.mako` to Vue. This is just a way to mount Vue inside `.mako` file.

Comment (OlegZharkov) [reply client/src/components/DatasetInformation/mount.js]: @dannon but you're right, I can do this without jquery

**Comment by OlegZharkov** [comment]
thank you @dannon!

**Pull request merged**